salvo-infra/azp-salvo-asg/iam.tf

```diff
@@ -0,0 +1,104 @@
+# Creates an IAM Role where instances themselves can then detach themselves from ASG.
```

Comment:
nit: feels like removing the first "themselves" reads better.

Reply:
Done in d45c7cca4b8665bb1c2ac197367cdedab4a21f64.